src/main/java/org/sagebionetworks/web/client/widget/table/v2/schema/ColumnModelsEditorV2Widget.java

```diff
@@ -0,0 +1,40 @@
+package org.sagebionetworks.web.client.widget.table.v2.schema;
+
+import com.google.gwt.user.client.ui.IsWidget;
+import com.google.gwt.user.client.ui.Widget;
+import com.google.inject.Inject;
+import java.util.List;
+import org.sagebionetworks.repo.model.EntityType;
+import org.sagebionetworks.repo.model.table.ColumnModel;
+import org.sagebionetworks.repo.model.table.ViewScope;
+
+public class ColumnModelsEditorV2Widget implements IsWidget {
```

Comment:
I considered extracting a common interface between ColumnModelsEditorWidget (v1) and ColumnModelsEditorV2Widget, which would have simplified initial changes to the classes that use the editor. It would have been a fairly complicated refactor because

1. ColumnModelsEditorWidget also implements its' view's presenter interface, which would be inappropriate to share
2. The v2 implementation handles more logic internally, so it wouldn't have as many methods, so we would have ended up needing to implement many no-op methods. 
3. I don't immediately know how we'd accomplish conditional dependency injection based on the experimental mode cookie.


I ultimately decided that it wasn't worth it for temporary code.

Reply:
Totally agree, no need to share a common interface.  In general, I think the more logic that's in new implementation, the less complicated the interface will be.

Reply:
The separation will make deletion of v1 that much easier.